src/openiss-reid-speed.sh

```diff
@@ -1,30 +1,30 @@
 #!/encs/bin/tcsh
 
-# Give job a name
-#SBATCH -J openiss-reid
+# Job name
+#SBATCH --job-name openiss-reid
 
-# Send an email when the job starts, finishes or if it is aborted.
+# Recieve email notifications when the job starts, finishes or fails.
 #SBATCH --mail-type=ALL
 
-# Specify the output file name
-#SBATCH -o openiss-reid-tfk.log
-
 # Set output directory to current
 #SBATCH --chdir=./
 
+# Specify the output file name
+#SBATCH -o openiss-reid-output-%A.log
+
 # Request Memory
-#SBATCH --mem=32G
+#SBATCH --mem=20G
 
 # Request CPU - comment this section if the job needs GPUs 
-##SBATCH -n 32
+##SBATCH -c 32
 
 # Request GPU - comment this section if the job needs CPUs and uncomment the previous section
 #SBATCH --gpus=1
 
 # Execute the script
 module load anaconda3/2023.03/default
-conda env create -f environment.yml -p /speed-scratch/$USER/reid-venv
-conda activate /speed-scratch/$USER/reid-venv
+conda env create -f environment.yml -p ../reid-venv
+conda activate ../reid-venv
```

Comment:
This assumes `../reid-venv` is a valid path and is inside of the repo cloned sources. It may reside by accident in a home dir if the repo was cloned there. Perhaps maybe make it a configurable variable before the code begins.

Reply:
Changes to the path has been reverted.